source/CMakeLists.txt

```diff
@@ -35,55 +35,55 @@ add_library(
 )
 
 list(APPEND device_srcs
-    module_hamilt/src/nonlocal.cpp
-    module_hamilt/src/veff.cpp
-    module_hamilt/src/ekinetic.cpp
-    module_pw/src/pw_multi_device.cpp
-    module_hsolver/src/dngvd_op.cpp
-    module_hsolver/src/math_kernel.cpp
-    module_elecstate/src/elecstate_multi_device.cpp
-    module_psi/src/memory_psi.cpp
-    module_psi/src/device.cpp
-    src_pw/src/force_multi_device.cpp
-    src_pw/src/stress_multi_device.cpp
-    src_pw/src/wf_multi_device.cpp
-    src_pw/src/vnl_multi_device.cpp
-    module_base/src/math_multi_device.cpp
+  module_hamilt/kernels/nonlocal_op.cpp
```

Comment:
Does this modify also necessary for Makefile.Objects?

Reply:
Yes, fixed